data/migrations/20220117123847_case_mgmt_schema.js

```diff
@@ -1,7 +1,7 @@
 exports.up = function (knex) {
   return knex.schema
     .createTable('locations', (t) => {
-      t.increments('ID').notNullable().unique().primary();
```

Comment:
I believe you are not supposed to change an existing migration, but rather should update an existing table with a new migration.

Reply:
That's one way of doing it. Same end result

Reply:
Got it thanks!